lms/djangoapps/discussion/config/settings.py

```diff
@@ -0,0 +1,22 @@
+"""
+Discussion settings.
+"""
+from django.conf import settings
+
+# .. toggle_name: FEATURES['ENABLE_FORUM_DAILY_DIGEST']
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: True
+# .. toggle_description: Set to True to enable forum notification features.
+# .. toggle_category: discussion
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2020-03-09
+# .. toggle_expiration_date: None
+# .. toggle_tickets: None
+# .. toggle_status: supported
+# .. toggle_warnings: None
+ENABLE_FORUM_DAILY_DIGEST = 'enable_forum_daily_digest'
```

Comment:
This is often a contrived way to give something to annotate for Django Settings.  What is the best practice?  See the encapsulation comment below.